RELEASES.md

```diff
@@ -0,0 +1,44 @@
+# Release Process
+
+This section describes the release processes for tracking, preparing, and creating new L3AF releases. This includes information around the release cycles and guides for developers responsible for preparing upcoming stable releases.
+
+## Active development
+
+Active development is happening on the `main` branch, and a new version can be released from it in every 10-12 weeks.
+
+## Release Tracking
+
+Feature work for upcoming releases is tracked through GitHub Projects.
```

Comment:
By "GitHub Projects" do you mean https://github.com/l3af-project/l3af-arch/projects?type=beta which currently shows 0 entries?  Or something else?  Can you make "GitHub Projects" be a link?